src/legacy/core_plugins/kibana/common/field_formats/types/duration.ts

```diff
@@ -68,16 +69,16 @@ export function createDurationFormat(FieldFormat) {
       return {
         inputFormat: DEFAULT_INPUT_FORMAT.kind,
         outputFormat: DEFAULT_OUTPUT_FORMAT.method,
-        outputPrecision: DEFAULT_OUTPUT_PRECISION
+        outputPrecision: DEFAULT_OUTPUT_PRECISION,
       };
     }
-    _convert(val) {
+    _convert(val: number) {
       const inputFormat = this.param('inputFormat');
       const outputFormat = this.param('outputFormat');
       const outputPrecision = this.param('outputPrecision');
       const human = this.isHuman();
       const prefix = val < 0 && human ? 'minus ' : '';
-      const duration = parseInputAsDuration(val, inputFormat);
+      const duration = parseInputAsDuration(val, inputFormat) as any;
```

Comment:
We have types for monent duration. you can use it

Reply:
Used. Thanks